src/pytest_codspeed/plugin.py

```diff
@@ -228,7 +228,10 @@ def pytest_runtest_protocol(item: pytest.Item, nextitem: pytest.Item | None):
         assert plugin.lib is not None
         runtest_call = pytest.CallInfo.from_call(
             lambda: _run_with_instrumentation(
-                plugin.lib, item.nodeid, item.config, item.runtest
+                plugin.lib,
+                item.nodeid,
+                item.config,
+                lambda: ihook.pytest_runtest_call(item=item),
```

Comment:
After digging some more I found we can use `pytest_pyfunc_call` instead of `pytest_runtest_protocol` (`runtest` invokes this hook, see https://github.com/pytest-dev/pytest/blob/5bb1363435a8cb3e2010505dbeb1e015c36beed6/src/_pytest/python.py#L1762-L1764)

I got the tests to pass locally but I may be missing something so let me know if you don't think this is the right thing to do